roles/ceph-osd/tasks/system_tuning.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: disable osd directory parsing by updatedb
+  command: updatedb -e /var/lib/ceph
+  changed_when: false
+  failed_when: false
+
+- name: install dependencies
+  apt: pkg="hugepages"
```

Comment:
For completeness, please make this installation conditional on `ceph_osd.disable_transparent_hugepage` as well.